digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala

```diff
@@ -391,6 +391,19 @@ Returns empty result as Json message, not as page not found
     }
   }
 
+  private def getRoadLinksFromVVH(municipalities: Set[Int], withRoadAddress: String)(bbox: String): Seq[Seq[Map[String, Any]]] = {
+    val boundingRectangle = constructBoundingRectangle(bbox)
+    validateBoundingBox(boundingRectangle)
+    val roadLinks = withRoadAddress match {
+      case "true" => roadLinkService.withRoadAddress(roadLinkService.getRoadLinksFromVVH(boundingRectangle, municipalities))
+      case _ => roadLinkService.getRoadLinksFromVVH(boundingRectangle, municipalities)
+    }

```

Comment:
Works like you have, but it's more pretty if you modify the withRoadAddress to Boolean type and instead of the case you use something like that:

val roadLinkSeq = roadLinkService.getRoadLinksFromVVH(boundingRectangle, municipalities)
val roadLinks = if (withRoadAddress) roadLinkService.withRoadAddress(roadLinkSeq) else roadLinkSeq